HIP/hip-904.md

```diff
@@ -7,11 +7,12 @@ requested-by: DEXs, Wallets, External Web3 Users. Notably Reality+, Liithos, Dro
 type: Standards Track
 category: Core
 needs-council-approval: Yes
-status: Accepted
+status: Final
+release: v0.54.0
```

Comment:
Should be v0.56 as that's when the system contract functions went in.
Agreed it's 0.54 for when HAPI functionality was completed but it might be confusing if users test this using local node or solo 😄

Reply:
That makes sense. I copied it from the previous PR.